Assets/HoloToolkit/Input/Scripts/Gaze/GazeManager.cs

```diff
@@ -281,6 +282,29 @@ private void RaycastUnityUI()
             }
         }
 
+        public override bool TryGetPosition(uint sourceId, out Vector3 position)
+        {
+            position = Vector3.zero;
```

Comment:
These are part of the 'BaseInputSource' abstract methods and properties. They have to be implemented although I can't find where it's used. I copied these implementations from 'GestureInput'.